daqWheel/src/main.cpp

```diff
@@ -0,0 +1,65 @@
+#include <Arduino.h>
+#include "daqWheel.cpp"
```

Comment:
I'm not as familiar with cpp guidelines, but is this seen as good practice to include the cpp file vs breaking it into a cpp and header file and including the header file?